src/wasm/kotlin/src/arcs/Particle.kt

```diff
@@ -39,16 +39,17 @@ abstract class Particle : WasmObject() {
         onHandleSync(handle, toSync.isEmpty())
     }
 
-    open fun onHandleUpdate(handle: Handle) {}
-    open fun onHandleSync(handle: Handle, allSynced: Boolean) {}
+    open fun init() {}
```

Comment:
Instead of an empty block, you can just do `= Unit`
```suggestion
    open fun init() = Unit
```